### ISSUE TITLE
Roslyn isn't generating files properly

### ISSUE THREAD

**Issue by WoLfulus** [sschmid/Entitas-CSharp#493]
I've been playing around a bit, trying to integrate the new generator into my project, but there's something wrong with the Roslyn generator. I don't really know how to explain, because it happens, and sometimes it doesn't.

I've recorded a video to show the behavior, but at first try both generators seems to have generated the files successfully, and I as you can see on the video I just started clicking everything trying to understand why it worked (I was excpecting it didn't) :P 

But doing some refactoring and changing files here and there, it broke at the end of the video.

https://www.youtube.com/watch?v=BoKB0mW826U

I stopped the recording before trying with old generator again, but it did work after fixing compilation errors, and after trying the roslyn generator, the old "Aloha" component got generated again (which is strange because the file doesn't even exists anymore).

The setup process was exactly what I saw from the guides/tutorial: import from asset store, put updated files from releases section, hit auto-import, hit generate (this worked fine - broke after playing around a little bit creating and refactoring components and systems)

**Comment by sschmid** [comment]
Hey, 

### tl;dr;
- Use the Unity code generator when working with the default reflection-based generator
- Use the standalone generator in server mode with roslyn

from the Video I saw, you're using the code generator in server mode. This mode was introduced to speed up the process when you use the roslyn plugin in. When using the reflection-based code generator the simplest thing is to use the Unity menu button.
Also, unfortunately once an assembly is loaded I can't really unload it, that is why server mode in combination with the reflection-based generator won't work.

**Comment by WoLfulus** [comment]
I can't confirm it now since I'm not home, but I'm pretty sure I was only using the server generator when I saw it behaving like this... I used both generators only to demonstrate that the generator server doesn't detect changes, while the reflection based (green button) generates files successfully.

I was expecting the inverse behavior because I had compilation errors (reflection based generating files based on the last compilation) and generator server (roslyn) generating files based on source contents.

I'll confirm later, but even restarting the server and trying to regenerate doesn't make it detect changes to the source files (even directly running `entitas gen`)

The behaviors seems really confusing and I'm pretty sure the auto-import didn't even detect the Roslyn generator (which might explain the odd behavior) and is using reflection based instead.

How `Entitas.properties` should look like for Roslyn generator?

**Comment by WoLfulus** [comment]
I've deleted the whole thing and imported the asset again. It was probably caused by me trying to use the updated version of GitHub with Asset Store Roslyn.

Any thoughts on a better way to get the updated versions faster than Asset Store? :P
